src/test/java/hw1/DataProviders.java

```diff
@@ -0,0 +1,29 @@
+package hw1;
+
+import org.testng.annotations.DataProvider;
+
+public class DataProviders {
+    @DataProvider(name = "double values")
+    public static Object[][] correctDoubleValuesData() {
+        return new Object[][]{
+                {Double.MAX_VALUE, Double.MAX_VALUE},
```

Comment:
DataProvider that provides only input values in combination with an etalon solution is used sometimes and is called Generator Driven Property Checks. But in my practice providers that provide expected results, as well as arguments, are used wider. This can be critical for computation hard processes (in terms of memory, time). 

So I would ask you to include results in DataProvider's values

Reply:
Done.